docs/_includes/head.html

```diff
@@ -3,7 +3,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
-  <link rel="stylesheet" href="{{ "/assets/css/style.css" | relative_url }}">
+  <webLink rel="stylesheet" href="{{ "/assets/css/style.css" | relative_url }}">
```

Comment:
Please make sure not to change `link` to `webLink`, else the HTML will not be rendered properly

Reply:
Done!